packages/backup-restore-operator/charts/examples/s3values.yaml

```diff
@@ -0,0 +1,14 @@
+image: 
+  repository: rancher/backup-restore-operator
+  tag: v0.0.1-rc4
+debug: false            # Enable debug logging in controller
+# Set as empty object {} if no s3 bucket needs to be configured for the operator
+s3: 
+  credentialSecretName: creds
+  region: us-west-2
+  bucketName: rajashree-backup-test
```

Comment:
Assume this has to change to replace your username? :smile:

Reply:
yeah that was in the examples folder for QA, I'll remove that examples folder 😅 And I think I have to update the commented section in actual values.yaml too